src/bin/console/console.cpp

```diff
@@ -535,34 +505,19 @@ int Console::visualize(const std::string& input) {
     }
   } else {
     // Compile the parse result
-    try {
-      plan = SQLPlanner::plan(parse_result);
-    } catch (const std::exception& exception) {
-      out("Exception thrown while compiling query plan:\n  " + std::string(exception.what()) + "\n");
-      return ReturnCode::Error;
-    }
+    auto plan = _sql_pipeline->get_query_plan();
```

Comment:
Aren't we missing a potential exception here completely? In terms of, if there is an exception in `_sql_pipeline->get_query_plan()` it throws, yes, but it does not properly return the `ReturnCode::Error` for the Console anymore.

Reply:
Yep, will fix.